src/custom-widget.ts

```diff
@@ -1,28 +1,16 @@
 import * as shortid from "shortid";
-import {
-  ITagoIO,
-  ICallbackStart,
-  IError,
-  IData,
-  IVariable,
-  ICallbackRealtime,
-  ICallbackError,
-  ITagoVariables,
-  IEvent,
-  IMessage,
-} from "./interfaces";
-import { enableAutofill } from "./utils";
+import { autoFillRecords } from "./utils";
 
 declare global {
   interface Window {
     /**
      * Send and receive data from the widget and variables from TagoIO
      */
-    TagoIO: ITagoIO;
+    TagoIO: TTagoIO;
```

Comment:
This file has a lot of changed name for types and every function, parameter, and variable now has typings.